src/tests/framework/Framework_Controller_Configuration_004.test.js

```diff
@@ -0,0 +1,99 @@
+import {
+  getPluginConfiguration,
+  pluginDeactivate,
+  setPluginConfiguration,
+} from '../../commonMethods/controller'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Framework Controller Configuration - 004',
+  description: 'Set the plugin configuration and check whether it is set or not',
+  steps: [
+    {
+      description: 'Get Plugin Configuration and validate the result',
+      test() {
+        return getPluginConfiguration.call(this, constants.webKitBrowserPlugin)
+      },
+      validate(res) {
+        this.$data.write('webkitconfig', res)
+        if (res !== undefined && res !== null) {
+          return true
+        } else {
+          throw new Error('Configuration not available')
+        }
+      },
+    },
+    {
+      description: 'Deactivate Webkit Browser',
+      test() {
+        let newConfig = this.$data.read('webkitconfig')
+        newConfig.fps = 'hello'
```

Comment:
I'm not sure if setting a FPS boolean to hello is safe, if WPEFramework decides to be stricter on JSON parsing this will immediately fail. Either change the boolean value or change something that is a string to hello (e.g. the start URL?).